spec/depth-enabled-photography.html

```diff
@@ -2160,121 +2160,417 @@
+                      },
+                      function(error) { console.log(error.message); });
+                },
+                function(error) { console.log(error.message); });
+          }
+
+          function depthEnhance() {
+            var before = new Date().getTime();
+            photoUtils.enhanceDepth(photo, 'high').then(
+                function(enhancedPhoto) {
+                  var after = new Date().getTime();
+                  var diff = after - before;
+                  enhancedPhoto.queryDepth().then(
+                      function(image) {
+                        // Render the depth enhanced photo image.
+                      },
+                      function(error) { console.log(error.message); });
+                },
+                function(error) { console.log(error.message); });
+          }
+
+          function depthResize() {
+            // Resize photo's depth image to 640px.
+            photoUtils.depthResize(photo, 640).then(
+                function(resizedPhoto) {
+                  resizedPhoto.queryDepth().then(
+                      function(image) {
+                        // Render the resized depth photo image.
+                      },
+                      function(error) { console.log(error.message); });
+                },
+                function(error) { console.log(error.message); });
+          }
+
+          function depthQuality() {
+            photoUtils.getDepthQuality(photo).then(
+                function(quality) {
+                  console.log('Depth qulity = ' + quality);
+                },
+                function(error) { console.log(error.message); });
+          }
+
+          function photoCrop() {
+            // Use originX, originY, width, height to define the bounding rectangle of the crop area.
+            photoUtils.photoCrop(photo, { x: originX, y: originY, w: width, h: height}).then(
+                function(croppedPhoto) {
+                  croppedPhoto.queryContainerImage().then(
+                      function(image) {
+                        // Render the cropped photo image.
+                      },
+                      function(error) { console.log(error.message); });
+                },
+                function(error) { console.log(error.message); });
+          }
+
+          function photoRotate() {
+            // Rotate photo by 90 degree clockwise.
+            photoUtils.photoRotate(photo, 90.0).then(
+                function(rotatedPhoto) {
+                  rotatedPhoto.queryContainerImage().then(
+                      function(image) {
+                        // Render the rotated photo image.
+                      },
+                      function(error) { console.log(error.message); });
+                },
+                function(error) { console.log(error.message); });
+          }
+        </pre>
+      </section>
+      <section>
+        <h3>
+          More Photo Process
```

Comment:
Would it be better to split this section to `Measurement`, `Motion Effect` and `Segmentation`? This example is a bit long. And in the future, it will be even longer.